docs/REST API/rest-api.md

```diff
@@ -947,7 +947,8 @@ None.
       "enable_end_user_authentication": false,
       "macos_setup_assistant": "path/to/config.json",
       "enable_release_device_manually": true
-    }
+    },
+    "client_url": "https://instance.fleet.com"
```

Comment:
Just to be sure, this is at `features.client_url` right?

Reply:
Hey @jacobshandling, I think no. We want this to be in the `mdm` object. So it would be `mdm.client_url`.

Reply:
Right, realized that but for some reason my comment didn't post. Thanks for the clarification:
<img width="817" alt="Screenshot 2024-10-08 at 1 59 12 PM" src="https://github.com/user-attachments/assets/a6bf2192-6739-4064-b723-606e9d44a3cb">